test/views/Delta.spec.js

```diff
@@ -226,6 +229,181 @@ describe('Delta', function () {
       });
 
     });
+
+    describe('using groupedTimeSeriesData', function () {
```

Comment:
Does it need another test(s) to confirm the formatted values are being generated, as well as the keys?